src/inmanta/logging.py

```diff
@@ -585,13 +585,7 @@ def clean_instance(cls, root_handlers_to_remove: Optional[abc.Sequence[logging.H
         By default, this method removes and closes all root handlers from the logging framework. If the
         root_handlers_to_remove argument is not None, only the provided root handlers will be removed and closed.
         """
-        to_remove = root_handlers_to_remove if root_handlers_to_remove is not None else logging.root.handlers
-        for handler in to_remove:
-            # File-based handlers automatically re-open after close() if log records are written to them.
-            # As such, we explicitly remove the handler from the root logger here.
-            logging.root.removeHandler(handler)
-            handler.flush()
-            handler.close()
```

Comment:
Shouldn't we remove the `root_handlers_to_remove` from the signature + docstring?

Reply:
I have been considering that, but this method is annotated with `@stable_api`, so we can't if we want to keep it backwards compatible. Although, I think this parameter is not used outside of inmanta-core.